pkg/constants/constants.go

```diff
@@ -78,4 +81,7 @@ const (
 	DefaultReloaderImage           = "connecteverything/nats-server-config-reloader"
 	DefaultReloaderImageTag        = "0.2.2-v1alpha2"
 	DefaultReloaderImagePullPolicy = "IfNotPresent"
+	DefaultMetricsImage            = "ttjiaa/prometheus-nats-exporter"
```

Comment:
let's change to use this one instead for the default too: `synadia/prometheus-nats-exporter:0.1.0`